components/report-details/support/form.tsx

```diff
@@ -0,0 +1,289 @@
+"use client";
+import ConnectButton from "@/components/connect-button";
+import { Button } from "@/components/ui/button";
+import {
+	Form,
+	FormControl,
+	FormDescription,
+	FormField,
+	FormItem,
+	FormLabel,
+	FormMessage,
+} from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import { Switch } from "@/components/ui/switch";
+import { Textarea } from "@/components/ui/textarea";
+import { useFunding } from "@/contexts/funding-context";
+import {
+	type TransactionStatuses,
+	useHandleBuyFraction,
+} from "@/hooks/use-buy-fraction";
+import { useEthersProvider } from "@/hooks/use-ethers-provider";
+import { useEthersSigner } from "@/hooks/use-ethers-signer";
+import useSupportForm from "@/hooks/use-support-form";
+import { cn } from "@/lib/utils";
+import type { Report } from "@/types";
+import { HypercertExchangeClient } from "@hypercerts-org/marketplace-sdk";
+import { useQuery } from "@tanstack/react-query";
+import { AlertTriangle, CheckCircle, Loader2, Wallet2 } from "lucide-react";
+import { sepolia } from "viem/chains";
+import { useAccount, usePublicClient } from "wagmi";
+import TransactionStatus from "./transaction-status";
```

Comment:
This comment is more personal preference but we should have some order to import statements.
something like:

framework
3rd partry
libs
hoooks
components